qemu/tests/qemu_guest_agent.py

```diff
@@ -1326,6 +1326,125 @@ def gagent_check_with_fsfreeze(self, test, params, env):
+        self._open_session_list.append(session)
+        logging.info("Change guest-file related cmd to white list.")
+        self._change_bl(session)
+
+        error_context.context("Create and write content to temp file and"
+                              " non temp file.", logging.info)
+        session.cmd("echo 'hello world' > %s" % guest_temp_file)
+        session.cmd("echo 'hello world' > %s" % guest_file)
+
+        error_context.context("Set selinux policy to 'Enforcing' mode in"
+                              " guest.", logging.info)
+        if session.cmd_output("getenforce").strip() != "Enforcing":
+            session.cmd("setenforce 1")
+
+        error_context.context("Try to open the temp file with w+"
+                              " mode via guest agent in 'Enforcing'"
+                              " selinux policy.", logging.info)
+        result_check(guest_temp_file, "w+", "Enforcing")
+
+        error_context.context("Try to open the temp file with a+"
+                              " mode via guest agent in 'Enforcing'"
+                              " selinux policy.", logging.info)
+        result_check(guest_temp_file, "a+", "Enforcing")
+
+        error_context.context("Try to open the non temp file with w+"
+                              " mode via guest agent in 'Enforcing'"
+                              " selinux policy.", logging.info)
+        result_check(guest_file, "w+", "Enforcing")
+
+        error_context.context("Try to open the non temp file with a+"
+                              " mode via guest agent in 'Enforcing'"
+                              " selinux policy.", logging.info)
+        result_check(guest_file, "a+", "Enforcing")
+
+        error_context.context("Set selinux policy to 'Permissive' mode in"
+                              " guest.", logging.info)
+        session.cmd("setenforce 0")
+
+        error_context.context("Try to open the temp file with w+"
+                              " mode via guest agent in 'Permissive'"
+                              " selinux policy.", logging.info)
+        result_check(guest_temp_file, "w+", "Permissive")
+
+        error_context.context("Try to open the temp file with a+"
+                              " mode via guest agent in 'Permissive'"
+                              " selinux policy.", logging.info)
+        result_check(guest_temp_file, "a+", "Permissive")
+
+        error_context.context("Try to open the non temp file with w+"
+                              " mode via guest agent in 'Permissive'"
+                              " selinux policy.", logging.info)
+        result_check(guest_file, "w+", "Permissive")
+
+        error_context.context("Try to open the non temp file with a+"
+                              " mode via guest agent in 'Permissive'"
+                              " selinux policy.", logging.info)
+        result_check(guest_file, "a+", "Permissive")
+
+        session.cmd("setenforce 1")
```

Comment:
@xiagao For RHEL7,  Is default value enforcing? If no, this is a wrong change.

Reply:
it's enforcing by default.

Reply:
And it's temporary setting,and after system reboot, it will recovery.

Reply:
Then is this line meaningless? Cause you won't check anything after this line.